app/tests/spec/views/sign_up.js

```diff
@@ -690,8 +691,8 @@ define(function (require, exports, module) {
               testExpectTriggered(1, 'signup.submit');
             });
 
-            it('fails with the correct error', function () {
-              assert.isTrue(AuthErrors.is(failed, 'AGE_REQUIRED'));
+            it('display an error message', function () {
+              assert.isTrue(view.showValidationError.called);
```

Comment:
Since this no longer tests `failed`, the the `.fail(...` at line 670 can probably go away, since now, if there is an error, we want it returned to mocha to report a failure, whereas right now we expect an error, save it, and don't tell mocha there was an error.